tests/acl/test_stress_acl.py

```diff
@@ -1,50 +1,126 @@
 
 LOOP_TIMES_LEVEL_MAP = {
     'debug': 10,
-    'basic': 500,
-    'confident': 5000
+    'basic': 50,
+    'confident': 200
 }
 
 # Template json file used to test scale rules
 STRESS_ACL_TABLE_TEMPLATE = "acl/templates/acltb_test_stress_acl_table.j2"
 STRESS_ACL_RULE_TEMPLATE = "acl/templates/acltb_test_stress_acl_rules.j2"
-STRESS_ACL_BASH_TEMPLATE = "acl/templates/acltb_test_stress_acl.sh"
 STRESS_ACL_TABLE_JSON_FILE = "/tmp/acltb_test_stress_acl_table.json"
 STRESS_ACL_RULE_JSON_FILE = "/tmp/acltb_test_stress_acl_rules.json"
-STRESS_ACL_BASH_FILE = "/tmp/acltb_test_stress_acl.sh"
-STRESS_ACL_BASH_LOG_FILE = "/tmp/acltb_test_stress_acl.log"
 
 LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
 LOG_EXPECT_ACL_RULE_FAILED_RE = ".*Failed to create ACL rule.*"
 
+ACL_RULE_NUMS = 10
+
+
+@pytest.fixture(scope='module')
+def prepare_test_file(rand_selected_dut):
+    # Define a custom table type CUSTOM_TYPE by loading a json configuration
+    rand_selected_dut.template(src=STRESS_ACL_TABLE_TEMPLATE, dest=STRESS_ACL_TABLE_JSON_FILE)
+    rand_selected_dut.shell("sonic-cfggen -j {} -w".format(STRESS_ACL_TABLE_JSON_FILE))
+    # Copy acl rules
+    rand_selected_dut.template(src=STRESS_ACL_RULE_TEMPLATE, dest=STRESS_ACL_RULE_JSON_FILE)
+
 
 @pytest.fixture(scope='module')
-def setup_stress_acl_table(rand_selected_dut, tbinfo, duthost):
+def prepare_dst_ip(rand_selected_dut, tbinfo):
+    routes = namedtuple('routes', ['route', 'aspath'])
+    common_config = tbinfo['topo']['properties']['configuration_properties'].get('common', {})
+    iproute_info = get_ip_route_info(rand_selected_dut)
+    dst_prefix_list = []
+    route_prefix_len = get_route_prefix_len(tbinfo, common_config)
+    for route_prefix in iproute_info:
+        if "/{}".format(route_prefix_len) in route_prefix:
+            multipath = iproute_info[route_prefix][0].get('multipath', False)
+            if multipath:
+                out = iproute_info[route_prefix][0].get('path').split(' ')
+                aspath = out[1:]
+                entry = routes(route_prefix, ' '.join(aspath))
+                dst_prefix_list.append(entry)
+                break
+
+    route_to_send = dst_prefix_list[0].route
+    dst_ip_addr = route_to_send.strip('/{}'.format(route_prefix_len))
+
+    return dst_ip_addr
+
+
+def parse_interfaces(output_lines, pc_ports_map):
+    """
+    Parse the inerfaces from 'show ip route' into an array
+    """
+    route_targets = []
+    ifaces = []
+    output_lines = output_lines[3:]
+
+    for item in output_lines:
+        match = re.search(r"(Ethernet\d+|PortChannel\d+)", item)
+        if match:
+            route_targets.append(match.group(0))
+
+    for route_target in route_targets:
+        if route_target.startswith("Ethernet"):
+            ifaces.append(route_target)
+        elif route_target.startswith("PortChannel") and route_target in pc_ports_map:
+            ifaces.extend(pc_ports_map[route_target])
+
+    return ifaces
+
```

Comment:
We don't have to make it so complicated. For the northbound traffic, the dest port must be uplink ports, and for southbound traffic, the dest port must be downlink ports.

Reply:
Fixed